src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+    <form
+      action="#"
+      method="get"
+      data-qa="big"
+    >
+
+
+
+      <div class="wrap">
```

Comment:
better use label here
```suggestion
      <label class="wrapper">
```

Reply:
but it`s easier to make it with div because it`s block element, and label is inline